builder/ebs/test-fixtures/ssh-keys/rsa_sha2_only_server.pkr.hcl

```diff
@@ -0,0 +1,40 @@
+/*
+OpenSSH migrated the ssh-rsa key type, which historically used the ssh-rsa
+signature algorithm based on SHA-1, to the new rsa-sha2-256 and rsa-sha2-512 signature algorithms.
+
+Golang issues: https://github.com/golang/go/issues/49952
+
+See plugin issue: https://github.com/hashicorp/packer-plugin-amazon/issues/213
+*/
+
+data "amazon-ami" "autogenerated_1" {
+  filters = {
+    name                = "ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-*"
+    root-device-type    = "ebs"
+    virtualization-type = "hvm"
+  }
+  most_recent = true
+  owners      = ["099720109477"]
+  region      = "us-east-1"
+}
+
+locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
+
+source "amazon-ebs" "basic-example" {
+  ami_description = "Ubuntu 22.04 GoldenImage"
```

Comment:
Do you know this one tests what you need because of the ubuntu version?

Reply:
Yes, this particular version of ubuntu runs openssh version 8.9 which requires stronger ssh key algorithms, as of openssh 8.8. I added a comment to the HCL file. But maybe I should add one here?

Reply:
I think it looks good the way it is!!